tracer/src/Datadog.Trace/Ci/CiEnvironment/GitCommandGitInfoProvider.cs

```diff
@@ -0,0 +1,144 @@
+// <copyright file="GitCommandGitInfoProvider.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+using System.IO;
+using System.Linq;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.Ci.CiEnvironment;
+
+internal sealed class GitCommandGitInfoProvider : GitInfoProvider
```

Comment:
This one is essentially the "new" provider, right?

Reply:
yup